docs/maatregelen/index.md

```diff
@@ -1,71 +1,80 @@
-
-Deze maatregel geeft invulling aan het vereiste van 'non-discriminatie'.
-Er kunnen meerdere maatregelen zijn die invulling geven aan het vereiste van non-discriminatie.
-Het is ook denkbaar dat bepaalde maatregelen aan meerdere vereisten kunnen worden gekoppeld.
-Dit moet organisaties helpen om met hun beschikbare middelen effectief en gericht te komen tot een verantwoorde ontwikkeling en inzet van algoritmes en AI.
-
-Om tot een overzicht van de maatregelen te komen is naar een aantal waardevolle bronnen gekeken.
-Het [Toetsingskader Algoritmes](https://www.rekenkamer.nl/onderwerpen/algoritmes/documenten/publicaties/2024/05/15/het-toetsingskader-aan-de-slag) van de Algemene Rekenkamer en het [Onderzoekskader algoritmes](https://www.rijksoverheid.nl/documenten/rapporten/2023/07/11/onderzoekskader-algoritmes-adr-2023)van de Auditdienst Rijk zijn daar goede voorbeelden van.
-Daarnaast is ook de kennis van experts binnen en buiten het team Algoritmekader toegevoegd om te kunnen duiden welke maatregelen in de praktijk kunnen worden getroffen om te voldoen aan de vereisten.
- 
-## Standaarden
-Op nationaal, Europees en internationaal niveau wordt momenteel gewerkt aan de totstandkoming van standaarden.
-Deze standaarden worden opgesteld door gespecialiseerde organisaties (NEN, JTC21 en ISO) en ondergaan een uitgebreid proces.
-Dit proces zorgt uiteindelijk voor brede consensus voor de standaarden.
-Het naleven van deze standaarden door organisaties levert een vermoeden van conformiteit op, ofwel, door het naleven van standaarden is de kans groot dat organisaties voldoen aan de vereisten.
-De standaarden worden daarom een waardevolle bron voor het Algoritmekader.
-Waar mogelijk zullen de standaarden onderdeel worden van het Algoritmekader en tot uiting komen in de laag van 'maatregelen'.
-Er zal een verwijzing worden gemaakt naar de betreffende standaard.
-Hierover moeten nog nadere afspraken worden gemaakt met deze standaardisatieorganisaties.
-
-## Overzicht van maatregelen
-
-Onderstaand volgt een overzicht van maatregelen per fase van de levenscyclus. 
-
-??? simple "Probleemanalyse"
-
-    <!-- list_maatregelen_1 levenscyclus/probleemanalyse -->
-
-??? simple "Ontwerp"
-
-    <!-- list_maatregelen_1 levenscyclus/ontwerp -->
-
-??? simple "Dataverkenning en datapreparatie"
-
-    <!-- list_maatregelen_1 levenscyclus/dataverkenning-en-datapreparatie -->
-
-??? simple "Ontwikkelen"
-
-    <!-- list_maatregelen_1 levenscyclus/ontwikkelen -->
-
-??? simple "Verificatie en validatie"
-
-    <!-- list_maatregelen_1 levenscyclus/verificatie-en-validatie -->
-    
-??? simple "Implementatie"
-
-    <!-- list_maatregelen_1 levenscyclus/implementatie -->
-    
-??? simple "Monitoring en beheer"
-
-    <!-- list_maatregelen_1 levenscyclus/monitoring-en-beheer -->
-    
-??? simple "Uitfaseren"
-
-    <!-- list_maatregelen_1 levenscyclus/uitfaseren -->
-
+# Maatregelen
```

Comment:
Hier ook 'aanbevolen maatregelen'?

Reply:
Mijn idee was om 'aanbevolen' alleen toe te voegen aan selecties van maatregelen. Bijvoorbeeld de aanbevolen maatregelen voor een specifieke eis, of voor een inkoopadviseur. Het overzicht van alle maatregelen lijkt me lastig aan te bevelen, omdat dan nog onduidelijk is om welke situatie het gaat.